src/network/clientpackethandler.cpp

```diff
@@ -122,7 +123,13 @@ void Client::handleCommand_AuthAccept(NetworkPacket* pkt)
 					<< m_recommended_send_interval<<std::endl;
 
 	// Reply to server
-	NetworkPacket resp_pkt(TOSERVER_INIT2, 0);
+	std::string lang = gettext("LANG_CODE");
+	if (lang == "LANG_CODE") {
+		lang = "";
+	}
+	lang = "fr";
+	NetworkPacket resp_pkt(TOSERVER_INIT2, 1 + lang.size());
```

Comment:
The length of the packet won't actually be 1 + lang.size().  I'd just leave out the length, but if you want to have it there, the correct length is probably 2 + lang.size().